tls/extensions/s2n_client_key_share.c

```diff
@@ -308,9 +308,9 @@ static int s2n_client_key_share_recv_pq_hybrid(struct s2n_connection *conn, stru
         return S2N_SUCCESS;
     }
 
-    struct s2n_kem_group_params *client_params = &conn->kex_params.client_kem_group_params;
+    struct s2n_kem_group_params *current_best_client_params = &conn->kex_params.client_kem_group_params;
 
-    const struct s2n_kem_group *kem_group = NULL;
+    const struct s2n_kem_group *new_best_match = NULL;
```

Comment:
I don't think these name changes are an improvement. They make sense in the context of this PR and the changes you're making to s2n_server_key_share.c, but they don't really make sense if you're only looking at s2n_client_key_share.c. Like, at this point the params are client_params, they're the params offered by the client. We just may decide not to use them later. "best" only comes up elsewhere in the code.

Reply:
Fixed